### PR TITLE
New script Command: add devhints to web searches

### PR DESCRIPTION
**Pull request by AsterYujano**
## Description

Add devhints.io to searches, with a search argument. Very convenient to get cheatsheets.

## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/web-searches/search-devhints.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.author Francois
```
Comment (unnamedd) [inline commands/web-searches/search-devhints.sh:4]: You may want to add some URL to your social medias:
```suggestion
# @raycast.author Francois
# @raycast.authorURL https://github.com/AsterYujano
```

File: commands/web-searches/search-devhints.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (unnamedd) [inline commands/web-searches/search-devhints.sh:1]: Will make sense rename this file to `search-in-devhints.sh`, to follow the pattern already used for the Script Commands which has title starting with `Search in`.

**Review by unnamedd — COMMENTED** [review]
Hi @AsterYujano, 
nothing huge to comment here, just the comment regarding an URL for your social medias.

And to finish, you don't need to change the commands/README.md, it is auto generated.

File: commands/web-searches/search-devhints.sh (hunk 2 above)
Comment (AsterYujano) [reply commands/web-searches/search-devhints.sh:1]: Hey! 

I was not sure as some other files do not follow the pattern (bilibili, github, etc), but I will push a commit doing so. 

Have a nice day,

PS: I love Raycast, sooo powerful :)

**Pull request merged**